drivers/SmartThings/matter-rvc/src/init.lua

```diff
@@ -343,6 +452,64 @@ local function rvc_operational_error_attr_handler(driver, device, ib, response)
   end
 end
 
+local function handle_rvc_operational_state_accepted_command_list(driver, device, ib, response)
+  device.log.info("handle_rvc_operational_state_accepted_command_list")
+  local cap_op_cmd = capabilities.robotCleanerOperatingState.commands
+  local OP_COMMAND_MAP = {
+    [clusters.RvcOperationalState.commands.Pause.ID] = cap_op_cmd.pause,
+    [clusters.RvcOperationalState.commands.Resume.ID] = cap_op_cmd.start,
+    [clusters.RvcOperationalState.commands.GoHome.ID] = cap_op_cmd.goHome
+  }
+  local supportedOperatingStateCommands = {}
+  for _, attr in ipairs(ib.data.elements) do
+    table.insert(supportedOperatingStateCommands, OP_COMMAND_MAP[attr.value].NAME)
+  end
+  device:set_field(OPERATING_STATE_SUPPORTED_COMMANDS, supportedOperatingStateCommands, { persist = true })
+
+  -- Get current run mode, current tag, current operating state
+  local current_run_mode = device:get_latest_state(
+    "runMode",
+    capabilities.mode.ID,
+    capabilities.mode.mode.NAME
+  )
+  local current_tag = 0xFFFF
+  local supported_run_modes = device:get_field(RUN_MODE_SUPPORTED_MODES) or {}
+  for _, mode in ipairs(supported_run_modes) do
+    if mode.label == current_run_mode then
+      current_tag = mode.tag
+      break
+    end
+  end
+  local operating_state = device:get_latest_state(
+    "main",
+    capabilities.robotCleanerOperatingState.ID,
+    capabilities.robotCleanerOperatingState.operatingState.NAME
+  )
+  local cap_op_enum = capabilities.robotCleanerOperatingState.operatingState
+  if operating_state ~= cap_op_enum.stopped.NAME and operating_state ~= cap_op_enum.running.NAME and
+     operating_state ~= cap_op_enum.paused.NAME and operating_state ~= cap_op_enum.seekingCharger.NAME and
+     operating_state ~= cap_op_enum.charging.NAME and operating_state ~= cap_op_enum.docked.NAME then
+      operating_state = "Error"
+  end
+
+  -- Set Supported Operating State Commands
+  local cap_op_cmd = capabilities.robotCleanerOperatingState.commands
```

Comment:
Are there circumstance where `supportedOperatingStateCommands` does not equal `cap_op_cmd`?

Or where `device:get_field(OPERATING_STATE_SUPPORTED_COMMANDS)` does not equal the `latest_value` of `supportedOperatingStateCommands`?